include/swift/Driver/FineGrainedDependencyDriverGraph.h

```diff
@@ -493,7 +493,12 @@ class ModuleDepGraph {
 
   template <typename Nodes>
   std::vector<const driver::Job *>
-  findJobsToRecompileWhenNodesChange(const Nodes &);
+  findJobsToRecompileWhenNodesChange(const Nodes &nodes) {
+    std::vector<ModuleDepGraphNode *> foundDependents;
+    for (ModuleDepGraphNode *n : nodes)
+      findPreviouslyUntracedDependents(foundDependents, n);
+    return jobsContaining(foundDependents);
+  }
```

Comment:
Looks like you were able to remove some template foo by moving the code to here. Is that right? If so, great!